assets/js/search-page.js

```diff
@@ -133,6 +134,26 @@ function snakeCaseSplitter (builder) {
   builder.pipeline.before(lunr.stemmer, snakeCaseFunction)
 }
 
+function elixirFunctionSplitter (builder) {
+  function splitter (token) {
+    const functionTokens = token
+      .toString()
+      .split(/\.|\//)
```

Comment:
For instance:

```
> "Enum.any?/2".split(/\.|\//)
[ 'Enum', 'any?', '2' ]
```

Reply:
Should we merge this splitter with the previous splitter? 🤔  And then split everything on `.` and `/`?

Reply:
Thanks for the suggestion. I pushed the changes.